js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1167,19 +1249,24 @@ const EditingController = modules.ViewController.inherit((function() {
         },
 
         editCell: function(rowIndex, columnIndex) {
+            return this._editCell(rowIndex, columnIndex);
+        },
+
+        _editCell: function(rowIndex, columnIndex, oldState) {
```

Comment:
The oldState argument is confusing a little bit. Isn't better to wrap parameters into objects? For instance: 

```js
_editCell: function(newParams, oldParams)
```
or

```js
_editCell: function(newState, oldState)
```

usage:
```js
const newParams = {
    rowIndex, 
    columnIndex
};
const oldParams = {
    rowIndex,
    columnIndex
};
this._editCell(newParams, oldParams);
```